### PR TITLE
Made CSS fixes

### PR DESCRIPTION
**Pull request by sanjam-deriv**

### CODE REVIEW

File: SchemaBody.tsx [hunk 1]
```diff
@@ -16,11 +16,12 @@ const Properities: React.FC<SchemaBodyProps> = ({ properties }) => {
...
   const CodeString: React.FC<CodeStringProps> = ({ description }) => {
       const highlightCode = description.split(" ").map((desc, index) => {
-          return (/`([^`]*)`/.test(desc)) ?
+          const x = desc[desc.length - 1]
+            return (/`([^`]*)`/.test(desc)) ?
               <span
                   className={`${styles.schemaRole} ${styles.schemaCode}`}
                   key={index}
-              >{`${desc.slice(1, desc.length - 1)}`}
+              >{`${x === "`" ? desc.slice(1, desc.length - 1) : desc.slice(1, desc.length - 2)}`}
```
Comment (hubert-deriv) [inline SchemaBody.tsx:24]: Can you explain me what this exactly does?

File: RequestJSONBox.module.scss [hunk 2]
```diff
@@ -41,7 +40,7 @@
     min-height: 200px;
     padding: 10px 12px;
     border: none;
-    color: var(--copy-color);
+    color: #5d5381;
```
Comment (hubert-deriv) [inline RequestJSONBox.module.scss:43]: Use color variable

**Review by hubert-deriv — CHANGES_REQUESTED** [review]

File: index.scss [hunk 3]
```diff
@@ -106,13 +106,13 @@ p {
...
-    color: var(--json-purple) !important;
+    color: var(--#e6db74) !important;
   }
   .token.punctuation {
-    color: var(--json-gold) !important;
+    color: var(--copy-color) !important;
   }
   .token.property {
-    color: var(--json-magenta) !important;
+    color: #f92672 !important;
```
[the remark this answers is not in the document]
Comment (sanjam-deriv) [reply index.scss:115]: The colors were supposed to be inverted to match production

File: RequestJSONBox.module.scss (hunk 2 above)
Comment (sanjam-deriv) [reply RequestJSONBox.module.scss:43]: the color used one time so is it necessary to create a variable for it?

File: SchemaBody.tsx (hunk 1 above)
Comment (sanjam-deriv) [reply SchemaBody.tsx:24]: it removes the '`' from reqid` in Schmea wrapper

**Comment by adam-deriv** [comment]
closing as https://github.com/binary-com/deriv-developers-portal/pull/93 introduces the code

**Pull request closed without merging**